### PR TITLE
Add license to EPSO

### PR DESCRIPTION
**Pull request by matentzn**
@phwegner

### CODE REVIEW

**Comment by phwegner** [comment]
@matentzn I added one just now

File: ontology/epso.md
Comment (cthoyt) [inline ontology/epso.md:10]: invalid. must be either 3.0 or 4.0

Comment (cthoyt) [reply ontology/epso.md:10]:
```suggestion
  label: CC BY 4.0
```

**Review by cthoyt — CHANGES_REQUESTED** [review]
Please make sure the label passes the schema (which isn't enforced until #1661 is merged) by accepting my suggestion

**Review by phwegner — APPROVED** [review]

**Review by phwegner — APPROVED** [review]

File: ontology/epso.md
Comment (phwegner) [reply ontology/epso.md:10]: Ok

**Comment by phwegner** [comment]
@cthoyt Okay I approved it

**Comment by cthoyt** [comment]
@phwegner that's not exactly what I meant. Since this is @matentzn's PR and I used the suggestion feature, Nico will have to accept my suggestion though the GitHub website, then it will make a commit on my behalf to his branch.

**Comment by phwegner** [comment]
@cthoyt Okay then I'll wait for you response

**Comment by phwegner** [comment]
@matentzn Do you have a URL that will later used for the Ontology (We need this for a paper)

**Review by nataled — APPROVED** [review]

**Pull request merged**
